build/configs/artik053/tc/defconfig

```diff
@@ -1427,6 +1427,7 @@ CONFIG_ITC_KERN_PTHREAD_LOOP_SIZE=100
 CONFIG_EXAMPLES_TESTCASE_NETWORK=y
 CONFIG_TC_NET_ALL=y
 CONFIG_TC_NET_SOCKET=y
+CONFIG_TC_NET_PKT=y
```

Comment:
Is this enough to enable only for artik? How about imxrt, rtl8721 and so on?

Reply:
done